### PR TITLE
Feature/show loading bar without cover

### PR DESCRIPTION
**Pull request by hutnikau**

### CODE REVIEW

**Comment by krampstudio** [comment]
Could you please describe the goal of this PR : what the issue, what to expected, what do you solve, may be a screenshot, etc.

**Comment by hutnikau** [comment]
I need to implement ability to cancel request of authorization of delivery execution. Test taker must be able to cancel it during waiting of authorization:
image
While test taker is on "awaiting" screen he must be able to click on "Cancel" button, and also the loader (sliding orange stripe) should be shown as sign that something is in progress.
Current implementation of loading bar does not allow to use GUI because of cover html element on top of page.

File: views/js/layout/loading-bar.js [hunk 1]
```diff
@@ -45,40 +56,44 @@ define(['jquery'],
...
+        }
+
+        $win.on('scroll.loadingbar', function () {
+            updateHeight();
         });
 
         return {
-            start: function () {
-                if($loadingBar.hasClass('loading')) {
+            start: function (covered) {
```
Comment (krampstudio) [inline views/js/layout/loading-bar.js]: Please document the function, especially the meaning of the new parameter `cover`

File: views/js/layout/loading-bar.js [hunk 2]
```diff
@@ -45,40 +56,44 @@ define(['jquery'],
...
+                if (covered === undefined) {
```
Comment (krampstudio) [inline views/js/layout/loading-bar.js]: By convention, we use `typeof covered === 'undefined'`

**Comment by hutnikau** [comment]
@krampstudio i have made requested changes and solved conflicts. Please review it again.

File: views/js/layout/loading-bar.js [hunk 3]
```diff
@@ -45,40 +56,48 @@ define(['jquery'],
...
+        $win.on('scroll.loadingbar', function () {
+            updateHeight();
         });
 
         return {
-            start: function () {
-                if($loadingBar.hasClass('loading')) {
+            /**
+             * Show loading bar
+             * @param bool covered - whether overlay HTML element should be added (disable GUI). Default - true
```
Comment (krampstudio) [inline views/js/layout/loading-bar.js]: The correct format is 
```js
* @param {Boolean} [covered = true] - whether overlay HTML element should be added (disable GUI).
```

**Comment by hutnikau** [comment]
@krampstudio, I have added unit test and fixed documentation. Please review it again.

**Comment by krampstudio** [comment]
Cool!

**Comment by krampstudio** [comment]
related to https://oat-sa.atlassian.net/browse/TAO-3862

**Pull request merged**